build.gradle

```diff
@@ -75,7 +79,7 @@ allprojects {
         // `disallow` in Java 18.
         // - https://openjdk.org/jeps/411
         // - https://bugs.openjdk.org/browse/JDK-8270380
-        if (project.ext.testJavaVersion >= 17) {
+        if (project.ext.testJavaVersion >= 17 || it.name == 'nativeImageTrace') {
```

Comment:
`testJavaVersion` is specified on per-project basis rather than on per-task basis, so I guess that's not possible? (`nativeImageTrace` is a task that is created for each project.) Please correct me if I'm wrong.